cmake/StaticBuild.cmake

```diff
@@ -130,7 +132,7 @@ if(BUILD_STATIC_GCC)
     set(STATIC_BASE_LIBRARIES "${STATIC_BASE_LIBRARIES} -Wl,-z -Wl,muldefs -Wl,-Bstatic -ltsan -Wl,-Bdynamic -ldl -lm")
   endif()
   if(WIN32)
-    set(STATIC_BASE_LIBRARIES "${STATIC_BASE_LIBRARIES} -lmingw32 -lgcc_eh -lgcc -lmoldname -lmingwex -lmsvcrt")
+    set(STATIC_BASE_LIBRARIES "${STATIC_BASE_LIBRARIES} -lmingw32 -lgcc_eh -lgcc -lmoldname -lmingwex -lmsvcrt -lpthread")
```

Comment:
pthread isn't used on Windows. Why was this needed?

Reply:
idk exactly, but without pthread linking of `convperf.exe` fails in my setup (idk what is it and/or why it is needed):
```
[ 92%] Building CXX object tests/perf/CMakeFiles/fbperf.dir/__/__/vncviewer/Surface_Win32.cxx.obj
[ 93%] Linking CXX executable fbperf.exe
[ 93%] Built target fbperf
Scanning dependencies of target decperf
[ 94%] Building CXX object tests/perf/CMakeFiles/decperf.dir/decperf.cxx.obj
[ 94%] Linking CXX executable decperf.exe
[ 94%] Built target decperf
Scanning dependencies of target convperf
[ 95%] Building CXX object tests/perf/CMakeFiles/convperf.dir/convperf.cxx.obj
[ 95%] Linking CXX executable convperf.exe
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/libgcc_eh.a(emutls.o):(.text+0x71): undefined reference to `pthread_key_create'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/libgcc_eh.a(emutls.o):(.text+0xad): undefined reference to `pthread_getspecific'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/libgcc_eh.a(emutls.o):(.text+0xf7): undefined reference to `pthread_once'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/libgcc_eh.a(emutls.o):(.text+0x157): undefined reference to `pthread_setspecific'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/libgcc_eh.a(emutls.o):(.text+0x21f): undefined reference to `pthread_setspecific'
collect2.exe: error: ld returned 1 exit status
make[2]: *** [tests/perf/CMakeFiles/convperf.dir/build.make:110: tests/perf/convperf.exe] Error 1
make[1]: *** [CMakeFiles/Makefile2:979: tests/perf/CMakeFiles/convperf.dir/all] Error 2
make: *** [Makefile:150: all] Error 2
```

building on Windows 10 using MSYS2 (Mingw64).